dart/lib/src/hub.dart

```diff
@@ -56,7 +52,7 @@ class Hub {
 
   /// Captures the event.
   Future<SentryId> captureEvent(
-    SentryEvent event, {
+    SentryEvent? event, {
```

Comment:
```suggestion
    SentryEvent event, {
```

it'd not make sense to accept a null SentryEvent, we'd drop it anyway